hooks/blueprint.pm

```diff
@@ -35,6 +35,7 @@ sub perform {
 	$blueprint->add_files(qw(
 		bosh-deployment/bosh.yml
 		bosh-deployment/uaa.yml
+		overlay/addons/prometheus-integration.yml
```

Comment:
Similar to hooks/blueprint, move this line to after the base file include on line 209